tests/pytorch/mock_sparse/test_sparse_matrix.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+import torch
+
+from dgl.mock_sparse import create_from_coo, create_from_csr, create_from_csc
+
+
```

Comment:
Let's add test for attributes too. `test_diag.py` is doing that.

Reply:
Added test for `device`. I think it cover all the attributes now.